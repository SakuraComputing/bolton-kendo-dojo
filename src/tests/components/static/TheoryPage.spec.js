import React from 'react';
import { shallow } from 'enzyme';
import TheoryPage from '../../../components/static/TheoryPage';

describe('Theory page', () => {
    it('should render correctly',() => {
       const wrapper = shallow(<TheoryPage />);
       expect(wrapper).toMatchSnapshot();
    });
});