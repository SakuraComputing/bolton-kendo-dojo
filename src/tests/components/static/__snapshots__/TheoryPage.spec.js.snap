// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Theory page should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TheoryPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="content-container"
      >
        <h1>
          General Definition
        </h1>
        Kendo (剣道 kendō, lit. "sword way") is a modern Japanese martial art, which descended from swordsmanship (kenjutsu) and uses bamboo swords (shinai) and protective armour (bōgu). Today, it is widely practiced within Japan and many other nations across the world. Kendo is an activity that combines martial arts practices and values with strenuous sport-like physical activity.
        <h1>
          All Japan Kendo Federation defintion of Kendo
        </h1>
        <div>
          <h2>
            The Concept of Kendo
          </h2>
          The concept of Kendo is to discipline the human character through the application of the principles of the Katana (sword).
          <h2>
            The Purpose of Practicing Kendo
          </h2>
          The purpose of practicing Kendo is: To mold the mind and body, To cultivate a vigorous spirit, And through correct and rigid training, To strive for improvement in the art of Kendo, To hold in esteem human courtesy and honor, To associate with others with sincerity, And to forever pursue the cultivation of oneself. This will make one be able: To love his/her country and society, To contribute to the development of culture And to promote peace and prosperity among all peoples. (The Concept of Kendo was established by All Japan Kendo Federation in 1975.)
          <h2>
            “The Mindset of Kendo Instruction” and its explanation
          </h2>
          <h3>
            The Significance of the Shinai
          </h3>
          For the correct transmission and development of Kendo, efforts should be made to teach the correct way of handling the shinai in accordance with the principles of the sword. Kendo is a way where the individual cultivates one’s mind (the self) by aiming for shin-ki-ryoku-itchi (unification of mind, spirit and technique) utilizing the shinai. The “shinai-sword” should be not only directed at one’s opponent but also at the self. Thus, the primary aim of instruction is to encourage the unification of mind, body and shinai through training in this discipline.
          <h3>
            Reiho - Etiquette
          </h3>
          When instructing, emphasis should be placed on etiquette to encourage respect for partners, and nurture people with a dignified and humane character. Even in competitive matches, importance is placed on upholding etiquette in Kendo. The primary emphasis should thus be placed on instruction in the spirit and forms of reiho (etiquette) so that the practitioner can develop a modest attitude to life, and realize the ideal of koken-chiai (the desire to achieve mutual understanding and betterment of humanity through Kendo.)
          <h3>
            Lifelong Kendo
          </h3>
          While providing instruction, students should be encouraged to apply the full measure of care to issues of safety and health, and to devote themselves to the development of their character throughout their lives. Kendo is a “way of life” that successive generations can learn together. The prime objective of instructing Kendo is to encourage the practitioner to discover and define their way in life through training in the techniques of Kendo. Thus, the practitioner will be able to develop a rich outlook on life and be able to put the culture of Kendo into use, thereby benefitting from its value in their daily lives through increased social vigour.
          <p />
          March 14, 2007 All Japan Kendo Federation * Translation of ‘Kendo Shido no Kokoro Gamae (The Mindset of Kendo Instruction)’ and its explanation in Japanese.
        </div>
      </div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            General Definition
          </h1>,
          "Kendo (剣道 kendō, lit. \\"sword way\\") is a modern Japanese martial art, which descended from swordsmanship (kenjutsu) and uses bamboo swords (shinai) and protective armour (bōgu). Today, it is widely practiced within Japan and many other nations across the world. Kendo is an activity that combines martial arts practices and values with strenuous sport-like physical activity.",
          <h1>
            All Japan Kendo Federation defintion of Kendo
          </h1>,
          <div>
            <h2>
              The Concept of Kendo
            </h2>
            The concept of Kendo is to discipline the human character through the application of the principles of the Katana (sword).
            <h2>
              The Purpose of Practicing Kendo
            </h2>
            The purpose of practicing Kendo is: To mold the mind and body, To cultivate a vigorous spirit, And through correct and rigid training, To strive for improvement in the art of Kendo, To hold in esteem human courtesy and honor, To associate with others with sincerity, And to forever pursue the cultivation of oneself. This will make one be able: To love his/her country and society, To contribute to the development of culture And to promote peace and prosperity among all peoples. (The Concept of Kendo was established by All Japan Kendo Federation in 1975.)
            <h2>
              “The Mindset of Kendo Instruction” and its explanation
            </h2>
            <h3>
              The Significance of the Shinai
            </h3>
            For the correct transmission and development of Kendo, efforts should be made to teach the correct way of handling the shinai in accordance with the principles of the sword. Kendo is a way where the individual cultivates one’s mind (the self) by aiming for shin-ki-ryoku-itchi (unification of mind, spirit and technique) utilizing the shinai. The “shinai-sword” should be not only directed at one’s opponent but also at the self. Thus, the primary aim of instruction is to encourage the unification of mind, body and shinai through training in this discipline.
            <h3>
              Reiho - Etiquette
            </h3>
            When instructing, emphasis should be placed on etiquette to encourage respect for partners, and nurture people with a dignified and humane character. Even in competitive matches, importance is placed on upholding etiquette in Kendo. The primary emphasis should thus be placed on instruction in the spirit and forms of reiho (etiquette) so that the practitioner can develop a modest attitude to life, and realize the ideal of koken-chiai (the desire to achieve mutual understanding and betterment of humanity through Kendo.)
            <h3>
              Lifelong Kendo
            </h3>
            While providing instruction, students should be encouraged to apply the full measure of care to issues of safety and health, and to devote themselves to the development of their character throughout their lives. Kendo is a “way of life” that successive generations can learn together. The prime objective of instructing Kendo is to encourage the practitioner to discover and define their way in life through training in the techniques of Kendo. Thus, the practitioner will be able to develop a rich outlook on life and be able to put the culture of Kendo into use, thereby benefitting from its value in their daily lives through increased social vigour.
            <p />
            March 14, 2007 All Japan Kendo Federation * Translation of ‘Kendo Shido no Kokoro Gamae (The Mindset of Kendo Instruction)’ and its explanation in Japanese.
          </div>,
        ],
        "className": "content-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "General Definition",
          },
          "ref": null,
          "rendered": "General Definition",
          "type": "h1",
        },
        "Kendo (剣道 kendō, lit. \\"sword way\\") is a modern Japanese martial art, which descended from swordsmanship (kenjutsu) and uses bamboo swords (shinai) and protective armour (bōgu). Today, it is widely practiced within Japan and many other nations across the world. Kendo is an activity that combines martial arts practices and values with strenuous sport-like physical activity.",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "All Japan Kendo Federation defintion of Kendo",
          },
          "ref": null,
          "rendered": "All Japan Kendo Federation defintion of Kendo",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2>
                The Concept of Kendo
              </h2>,
              "The concept of Kendo is to discipline the human character through the application of the principles of the Katana (sword).",
              <h2>
                The Purpose of Practicing Kendo
              </h2>,
              "The purpose of practicing Kendo is: To mold the mind and body, To cultivate a vigorous spirit, And through correct and rigid training, To strive for improvement in the art of Kendo, To hold in esteem human courtesy and honor, To associate with others with sincerity, And to forever pursue the cultivation of oneself. This will make one be able: To love his/her country and society, To contribute to the development of culture And to promote peace and prosperity among all peoples. (The Concept of Kendo was established by All Japan Kendo Federation in 1975.)",
              <h2>
                “The Mindset of Kendo Instruction” and its explanation
              </h2>,
              <h3>
                The Significance of the Shinai
              </h3>,
              "For the correct transmission and development of Kendo, efforts should be made to teach the correct way of handling the shinai in accordance with the principles of the sword. Kendo is a way where the individual cultivates one’s mind (the self) by aiming for shin-ki-ryoku-itchi (unification of mind, spirit and technique) utilizing the shinai. The “shinai-sword” should be not only directed at one’s opponent but also at the self. Thus, the primary aim of instruction is to encourage the unification of mind, body and shinai through training in this discipline.",
              <h3>
                Reiho - Etiquette
              </h3>,
              "When instructing, emphasis should be placed on etiquette to encourage respect for partners, and nurture people with a dignified and humane character. Even in competitive matches, importance is placed on upholding etiquette in Kendo. The primary emphasis should thus be placed on instruction in the spirit and forms of reiho (etiquette) so that the practitioner can develop a modest attitude to life, and realize the ideal of koken-chiai (the desire to achieve mutual understanding and betterment of humanity through Kendo.)",
              <h3>
                Lifelong Kendo
              </h3>,
              "While providing instruction, students should be encouraged to apply the full measure of care to issues of safety and health, and to devote themselves to the development of their character throughout their lives. Kendo is a “way of life” that successive generations can learn together. The prime objective of instructing Kendo is to encourage the practitioner to discover and define their way in life through training in the techniques of Kendo. Thus, the practitioner will be able to develop a rich outlook on life and be able to put the culture of Kendo into use, thereby benefitting from its value in their daily lives through increased social vigour.",
              <p />,
              "March 14, 2007 All Japan Kendo Federation * Translation of ‘Kendo Shido no Kokoro Gamae (The Mindset of Kendo Instruction)’ and its explanation in Japanese.",
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "The Concept of Kendo",
              },
              "ref": null,
              "rendered": "The Concept of Kendo",
              "type": "h2",
            },
            "The concept of Kendo is to discipline the human character through the application of the principles of the Katana (sword).",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "The Purpose of Practicing Kendo",
              },
              "ref": null,
              "rendered": "The Purpose of Practicing Kendo",
              "type": "h2",
            },
            "The purpose of practicing Kendo is: To mold the mind and body, To cultivate a vigorous spirit, And through correct and rigid training, To strive for improvement in the art of Kendo, To hold in esteem human courtesy and honor, To associate with others with sincerity, And to forever pursue the cultivation of oneself. This will make one be able: To love his/her country and society, To contribute to the development of culture And to promote peace and prosperity among all peoples. (The Concept of Kendo was established by All Japan Kendo Federation in 1975.)",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "“The Mindset of Kendo Instruction” and its explanation",
              },
              "ref": null,
              "rendered": "“The Mindset of Kendo Instruction” and its explanation",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "The Significance of the Shinai",
              },
              "ref": null,
              "rendered": "The Significance of the Shinai",
              "type": "h3",
            },
            "For the correct transmission and development of Kendo, efforts should be made to teach the correct way of handling the shinai in accordance with the principles of the sword. Kendo is a way where the individual cultivates one’s mind (the self) by aiming for shin-ki-ryoku-itchi (unification of mind, spirit and technique) utilizing the shinai. The “shinai-sword” should be not only directed at one’s opponent but also at the self. Thus, the primary aim of instruction is to encourage the unification of mind, body and shinai through training in this discipline.",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Reiho - Etiquette",
              },
              "ref": null,
              "rendered": "Reiho - Etiquette",
              "type": "h3",
            },
            "When instructing, emphasis should be placed on etiquette to encourage respect for partners, and nurture people with a dignified and humane character. Even in competitive matches, importance is placed on upholding etiquette in Kendo. The primary emphasis should thus be placed on instruction in the spirit and forms of reiho (etiquette) so that the practitioner can develop a modest attitude to life, and realize the ideal of koken-chiai (the desire to achieve mutual understanding and betterment of humanity through Kendo.)",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Lifelong Kendo",
              },
              "ref": null,
              "rendered": "Lifelong Kendo",
              "type": "h3",
            },
            "While providing instruction, students should be encouraged to apply the full measure of care to issues of safety and health, and to devote themselves to the development of their character throughout their lives. Kendo is a “way of life” that successive generations can learn together. The prime objective of instructing Kendo is to encourage the practitioner to discover and define their way in life through training in the techniques of Kendo. Thus, the practitioner will be able to develop a rich outlook on life and be able to put the culture of Kendo into use, thereby benefitting from its value in their daily lives through increased social vigour.",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "p",
            },
            "March 14, 2007 All Japan Kendo Federation * Translation of ‘Kendo Shido no Kokoro Gamae (The Mindset of Kendo Instruction)’ and its explanation in Japanese.",
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="content-container"
        >
          <h1>
            General Definition
          </h1>
          Kendo (剣道 kendō, lit. "sword way") is a modern Japanese martial art, which descended from swordsmanship (kenjutsu) and uses bamboo swords (shinai) and protective armour (bōgu). Today, it is widely practiced within Japan and many other nations across the world. Kendo is an activity that combines martial arts practices and values with strenuous sport-like physical activity.
          <h1>
            All Japan Kendo Federation defintion of Kendo
          </h1>
          <div>
            <h2>
              The Concept of Kendo
            </h2>
            The concept of Kendo is to discipline the human character through the application of the principles of the Katana (sword).
            <h2>
              The Purpose of Practicing Kendo
            </h2>
            The purpose of practicing Kendo is: To mold the mind and body, To cultivate a vigorous spirit, And through correct and rigid training, To strive for improvement in the art of Kendo, To hold in esteem human courtesy and honor, To associate with others with sincerity, And to forever pursue the cultivation of oneself. This will make one be able: To love his/her country and society, To contribute to the development of culture And to promote peace and prosperity among all peoples. (The Concept of Kendo was established by All Japan Kendo Federation in 1975.)
            <h2>
              “The Mindset of Kendo Instruction” and its explanation
            </h2>
            <h3>
              The Significance of the Shinai
            </h3>
            For the correct transmission and development of Kendo, efforts should be made to teach the correct way of handling the shinai in accordance with the principles of the sword. Kendo is a way where the individual cultivates one’s mind (the self) by aiming for shin-ki-ryoku-itchi (unification of mind, spirit and technique) utilizing the shinai. The “shinai-sword” should be not only directed at one’s opponent but also at the self. Thus, the primary aim of instruction is to encourage the unification of mind, body and shinai through training in this discipline.
            <h3>
              Reiho - Etiquette
            </h3>
            When instructing, emphasis should be placed on etiquette to encourage respect for partners, and nurture people with a dignified and humane character. Even in competitive matches, importance is placed on upholding etiquette in Kendo. The primary emphasis should thus be placed on instruction in the spirit and forms of reiho (etiquette) so that the practitioner can develop a modest attitude to life, and realize the ideal of koken-chiai (the desire to achieve mutual understanding and betterment of humanity through Kendo.)
            <h3>
              Lifelong Kendo
            </h3>
            While providing instruction, students should be encouraged to apply the full measure of care to issues of safety and health, and to devote themselves to the development of their character throughout their lives. Kendo is a “way of life” that successive generations can learn together. The prime objective of instructing Kendo is to encourage the practitioner to discover and define their way in life through training in the techniques of Kendo. Thus, the practitioner will be able to develop a rich outlook on life and be able to put the culture of Kendo into use, thereby benefitting from its value in their daily lives through increased social vigour.
            <p />
            March 14, 2007 All Japan Kendo Federation * Translation of ‘Kendo Shido no Kokoro Gamae (The Mindset of Kendo Instruction)’ and its explanation in Japanese.
          </div>
        </div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1>
              General Definition
            </h1>,
            "Kendo (剣道 kendō, lit. \\"sword way\\") is a modern Japanese martial art, which descended from swordsmanship (kenjutsu) and uses bamboo swords (shinai) and protective armour (bōgu). Today, it is widely practiced within Japan and many other nations across the world. Kendo is an activity that combines martial arts practices and values with strenuous sport-like physical activity.",
            <h1>
              All Japan Kendo Federation defintion of Kendo
            </h1>,
            <div>
              <h2>
                The Concept of Kendo
              </h2>
              The concept of Kendo is to discipline the human character through the application of the principles of the Katana (sword).
              <h2>
                The Purpose of Practicing Kendo
              </h2>
              The purpose of practicing Kendo is: To mold the mind and body, To cultivate a vigorous spirit, And through correct and rigid training, To strive for improvement in the art of Kendo, To hold in esteem human courtesy and honor, To associate with others with sincerity, And to forever pursue the cultivation of oneself. This will make one be able: To love his/her country and society, To contribute to the development of culture And to promote peace and prosperity among all peoples. (The Concept of Kendo was established by All Japan Kendo Federation in 1975.)
              <h2>
                “The Mindset of Kendo Instruction” and its explanation
              </h2>
              <h3>
                The Significance of the Shinai
              </h3>
              For the correct transmission and development of Kendo, efforts should be made to teach the correct way of handling the shinai in accordance with the principles of the sword. Kendo is a way where the individual cultivates one’s mind (the self) by aiming for shin-ki-ryoku-itchi (unification of mind, spirit and technique) utilizing the shinai. The “shinai-sword” should be not only directed at one’s opponent but also at the self. Thus, the primary aim of instruction is to encourage the unification of mind, body and shinai through training in this discipline.
              <h3>
                Reiho - Etiquette
              </h3>
              When instructing, emphasis should be placed on etiquette to encourage respect for partners, and nurture people with a dignified and humane character. Even in competitive matches, importance is placed on upholding etiquette in Kendo. The primary emphasis should thus be placed on instruction in the spirit and forms of reiho (etiquette) so that the practitioner can develop a modest attitude to life, and realize the ideal of koken-chiai (the desire to achieve mutual understanding and betterment of humanity through Kendo.)
              <h3>
                Lifelong Kendo
              </h3>
              While providing instruction, students should be encouraged to apply the full measure of care to issues of safety and health, and to devote themselves to the development of their character throughout their lives. Kendo is a “way of life” that successive generations can learn together. The prime objective of instructing Kendo is to encourage the practitioner to discover and define their way in life through training in the techniques of Kendo. Thus, the practitioner will be able to develop a rich outlook on life and be able to put the culture of Kendo into use, thereby benefitting from its value in their daily lives through increased social vigour.
              <p />
              March 14, 2007 All Japan Kendo Federation * Translation of ‘Kendo Shido no Kokoro Gamae (The Mindset of Kendo Instruction)’ and its explanation in Japanese.
            </div>,
          ],
          "className": "content-container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "General Definition",
            },
            "ref": null,
            "rendered": "General Definition",
            "type": "h1",
          },
          "Kendo (剣道 kendō, lit. \\"sword way\\") is a modern Japanese martial art, which descended from swordsmanship (kenjutsu) and uses bamboo swords (shinai) and protective armour (bōgu). Today, it is widely practiced within Japan and many other nations across the world. Kendo is an activity that combines martial arts practices and values with strenuous sport-like physical activity.",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "All Japan Kendo Federation defintion of Kendo",
            },
            "ref": null,
            "rendered": "All Japan Kendo Federation defintion of Kendo",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h2>
                  The Concept of Kendo
                </h2>,
                "The concept of Kendo is to discipline the human character through the application of the principles of the Katana (sword).",
                <h2>
                  The Purpose of Practicing Kendo
                </h2>,
                "The purpose of practicing Kendo is: To mold the mind and body, To cultivate a vigorous spirit, And through correct and rigid training, To strive for improvement in the art of Kendo, To hold in esteem human courtesy and honor, To associate with others with sincerity, And to forever pursue the cultivation of oneself. This will make one be able: To love his/her country and society, To contribute to the development of culture And to promote peace and prosperity among all peoples. (The Concept of Kendo was established by All Japan Kendo Federation in 1975.)",
                <h2>
                  “The Mindset of Kendo Instruction” and its explanation
                </h2>,
                <h3>
                  The Significance of the Shinai
                </h3>,
                "For the correct transmission and development of Kendo, efforts should be made to teach the correct way of handling the shinai in accordance with the principles of the sword. Kendo is a way where the individual cultivates one’s mind (the self) by aiming for shin-ki-ryoku-itchi (unification of mind, spirit and technique) utilizing the shinai. The “shinai-sword” should be not only directed at one’s opponent but also at the self. Thus, the primary aim of instruction is to encourage the unification of mind, body and shinai through training in this discipline.",
                <h3>
                  Reiho - Etiquette
                </h3>,
                "When instructing, emphasis should be placed on etiquette to encourage respect for partners, and nurture people with a dignified and humane character. Even in competitive matches, importance is placed on upholding etiquette in Kendo. The primary emphasis should thus be placed on instruction in the spirit and forms of reiho (etiquette) so that the practitioner can develop a modest attitude to life, and realize the ideal of koken-chiai (the desire to achieve mutual understanding and betterment of humanity through Kendo.)",
                <h3>
                  Lifelong Kendo
                </h3>,
                "While providing instruction, students should be encouraged to apply the full measure of care to issues of safety and health, and to devote themselves to the development of their character throughout their lives. Kendo is a “way of life” that successive generations can learn together. The prime objective of instructing Kendo is to encourage the practitioner to discover and define their way in life through training in the techniques of Kendo. Thus, the practitioner will be able to develop a rich outlook on life and be able to put the culture of Kendo into use, thereby benefitting from its value in their daily lives through increased social vigour.",
                <p />,
                "March 14, 2007 All Japan Kendo Federation * Translation of ‘Kendo Shido no Kokoro Gamae (The Mindset of Kendo Instruction)’ and its explanation in Japanese.",
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "The Concept of Kendo",
                },
                "ref": null,
                "rendered": "The Concept of Kendo",
                "type": "h2",
              },
              "The concept of Kendo is to discipline the human character through the application of the principles of the Katana (sword).",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "The Purpose of Practicing Kendo",
                },
                "ref": null,
                "rendered": "The Purpose of Practicing Kendo",
                "type": "h2",
              },
              "The purpose of practicing Kendo is: To mold the mind and body, To cultivate a vigorous spirit, And through correct and rigid training, To strive for improvement in the art of Kendo, To hold in esteem human courtesy and honor, To associate with others with sincerity, And to forever pursue the cultivation of oneself. This will make one be able: To love his/her country and society, To contribute to the development of culture And to promote peace and prosperity among all peoples. (The Concept of Kendo was established by All Japan Kendo Federation in 1975.)",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "“The Mindset of Kendo Instruction” and its explanation",
                },
                "ref": null,
                "rendered": "“The Mindset of Kendo Instruction” and its explanation",
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "The Significance of the Shinai",
                },
                "ref": null,
                "rendered": "The Significance of the Shinai",
                "type": "h3",
              },
              "For the correct transmission and development of Kendo, efforts should be made to teach the correct way of handling the shinai in accordance with the principles of the sword. Kendo is a way where the individual cultivates one’s mind (the self) by aiming for shin-ki-ryoku-itchi (unification of mind, spirit and technique) utilizing the shinai. The “shinai-sword” should be not only directed at one’s opponent but also at the self. Thus, the primary aim of instruction is to encourage the unification of mind, body and shinai through training in this discipline.",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Reiho - Etiquette",
                },
                "ref": null,
                "rendered": "Reiho - Etiquette",
                "type": "h3",
              },
              "When instructing, emphasis should be placed on etiquette to encourage respect for partners, and nurture people with a dignified and humane character. Even in competitive matches, importance is placed on upholding etiquette in Kendo. The primary emphasis should thus be placed on instruction in the spirit and forms of reiho (etiquette) so that the practitioner can develop a modest attitude to life, and realize the ideal of koken-chiai (the desire to achieve mutual understanding and betterment of humanity through Kendo.)",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Lifelong Kendo",
                },
                "ref": null,
                "rendered": "Lifelong Kendo",
                "type": "h3",
              },
              "While providing instruction, students should be encouraged to apply the full measure of care to issues of safety and health, and to devote themselves to the development of their character throughout their lives. Kendo is a “way of life” that successive generations can learn together. The prime objective of instructing Kendo is to encourage the practitioner to discover and define their way in life through training in the techniques of Kendo. Thus, the practitioner will be able to develop a rich outlook on life and be able to put the culture of Kendo into use, thereby benefitting from its value in their daily lives through increased social vigour.",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": "p",
              },
              "March 14, 2007 All Japan Kendo Federation * Translation of ‘Kendo Shido no Kokoro Gamae (The Mindset of Kendo Instruction)’ and its explanation in Japanese.",
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
